hack/release/chart-push.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -ex
+
+# github repo owner: e.g k8snetworkplumbingwg
+GITHUB_REPO_OWNER=${GITHUB_REPO_OWNER:-}
+# github api token with package:write permissions
+GITHUB_TOKEN=${GITHUB_TOKEN:-}
+# github tag e.g v1.2.3
+GITHUB_TAG=${GITHUB_TAG:-}
+
+BASE=${PWD}
+HELM_CHART=${BASE}/deployment/sriov-network-operator
+HELM_CHART_VERSION=${GITHUB_TAG#"v"}
+HELM_CHART_TARBALL="sriov-network-operator-${HELM_CHART_VERSION}.tgz"
+
+# make sure helm is installed
+set +e
+which helm
+if [ $? -ne 0 ]; then
+    echo "ERROR: helm must be installed"
+    exit 1
+fi
+set -e
+
+if [ -z "$GITHUB_REPO_OWNER" ]; then
+    echo "ERROR: GITHUB_REPO_OWNER must be provided as env var"
+    exit 1
+fi
+
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "ERROR: GITHUB_TOKEN must be provided as env var"
+    exit 1
+fi
+
+if [ -z "$GITHUB_TAG" ]; then
+    echo "ERROR: GITHUB_TAG must be provided as env var"
+    exit 1
+fi
+
+helm package ${HELM_CHART}
+helm registry login ghcr.io -u ${GITHUB_REPO_OWNER} -p ${GITHUB_TOKEN}
+helm push ${HELM_CHART_TARBALL} oci://ghcr.io/${GITHUB_REPO_OWNER}
```

Comment:
I think that the chart image name will be copied from the chart name. In our case this is a problem, because chart name is `sriov-network-operator` and we already have `sriov-network-operator` image in the GH registry (operator's controller)

Reply:
I am also with @ykulazhenkov here can you add 

```
oci://ghcr.io/${GITHUB_REPO_OWNER}-chart
```

Reply:
we should also add the tag here as the version no?

Reply:
> we should also add the tag here as the version no?

helm takes the chart version as the tag.

> oci://ghcr.io/${GITHUB_REPO_OWNER}-chart

we cant do it like that.

need to rename the name of the chart to : "sriov-network-operator-chart"

notice that we push to:  oci://ghcr.io/k8snetworkplumbingwg

helm then creates a container with the name of the chart and tags with the version of the chart then pushes it

Reply:
perhaps we can rename the sriov operator controller image from:

sriov-network-operator
to 
sriov-network-operator-controlller 

WDYT?

Reply:
ok I saw you changed the chart I prefer that instead of changing the operator name that has been running for a very long time.
also in an update, we will need logic to remove the old deployment. I prefer not to go that way